homework24/script.js

```diff
@@ -11,55 +11,80 @@ class Hamburger {
     _sauce = '';
     _price = 0;
     _calories = 0;
+
     constructor(size, stuffing) {
         this._size = size;
         this._stuffing = stuffing;
     };
 
+    getCalories(type) {
+        switch(type) {
+            case Hamburger.SIZE_SMALL:
+                return 20;
+                break;
+            case Hamburger.SIZE_LARGE:
+                return 40;
+                break;
+            case Hamburger.STUFFING_CHEESE: 
+                return 20;
+                break;
+            case Hamburger.STUFFING_POTATO:
+                return 10;
+                break;
+            case Hamburger.STUFFING_SALAD:
+                 return 5;
+                 break;
+            default:
+                return 0;
+                
+             
+        }
+    };
+    
+    getPrice(type) {
+        switch(type) {
+            case Hamburger.SIZE_SMALL:
+                return 50;
+                break;
+            case Hamburger.SIZE_LARGE:
+                return 100;
+                break;
+            case Hamburger.STUFFING_CHEESE: 
+                return 10;
+                break;
+            case Hamburger.STUFFING_POTATO:
+                return 15;
+                break;
+            case Hamburger.STUFFING_SALAD:
+                 return 20;
+                 break;
+            default:
+                return 0;   
+        }
+    }
+
     addTopping(topping) {
-        if(topping === 'sauce') {
+        if(topping === Hamburger.TOPPING_SAUCE) {
```

Comment:
you can also move these `Hamburger.TOPPING_SAUCE` to `getPrice` and `getCalories` 

```
if(topping === Hamburger.TOPPING_SAUCE) {
  this._price += this.getPrice(topping);
} else {
  // ...
}

```